pkg/coremodel/pluginmeta/coremodel.cue

```diff
@@ -142,7 +142,7 @@ seqs: [
 
 					// Required Grafana version for this plugin. Validated using
 					// https://github.com/npm/node-semver.
-					grafanaDependency: =~"^(<=|>=|<|>|=|~|\\^)?([0-9]+)(\\.[0-9x\\*]+)(\\.[0-9x\\*])?(\\s(<=|>=|<|=>)?([0-9]+)(\\.[0-9x]+)(\\.[0-9x]))?$"
+					grafanaDependency: =~"^(<=|>=|<|>|=|~|\\^)?([0-9]+)(\\.[0-9x\\*]+)(\\.[0-9x\\*]+)?(\\s(<=|>=|<|=>)?([0-9]+)(\\.[0-9x]+)(\\.[0-9x]))?$"
```

Comment:
Should this exactly match the `plugin.schema.json`? Seems it only adds the `+` for this one

Reply:
Oof! good point, copy paste... I will fix the plugins.schema.json one it should only add the `+`

Reply:
https://github.com/grafana/grafana/pull/54515/commits/bc4d2675c71672bc52f7cb4233549fcadefebc38